abstutil/src/logger.rs

```diff
@@ -16,7 +16,7 @@ impl Logger {
     pub fn setup() {
         #[cfg(target_arch = "wasm32")]
         {
-            console_log::init_with_level(log::Level::Debug).unwrap();
+            console_log::init_with_level(log::Level::Info).unwrap();
```

Comment:
I often spend some time engineering nice logging, but find it's too spammy for general purpose. I'd like to leave the spammy logging at debug level and bump up our default log level.

Relatedly, how would you feel about incorporating something like [env_logger](https://docs.rs/env_logger/0.8.2/env_logger/) so the user can adjust their log level without recompiling?